repo/packages/M/marathon/9/config.json

```diff
@@ -0,0 +1,378 @@
+{
+  "additionalProperties" : false,
+  "description" : "Marathon Universe properties",
+  "properties" : {
+    "application" : {
+      "additionalProperties" : false,
+      "description" : "Marathon app configuration properties.",
+      "properties" : {
+        "id" : {
+          "description" : "The ID of this marathon application name.",
```

Comment:
From #268:

> marathon should be Marathon. This description is a little confusing. How is this different to framework-name? We should stick to just using framework-name if possible since that would be consistent with our other packages.